Instructions/Exercises/03-prompt-engineering.md

```diff
@@ -59,12 +59,12 @@ Azure provides a web-based portal named **Azure AI Foundry portal**, that you ca
 
 Let's start by exploring some prompt engineering techniques in the Chat playground.
 
-1. In the **Playground** section, select the **Chat** page. The **Chat** playground page consists of a row of buttons and two main panels (which may be arranged right-to-left horizontally, or top-to-bottom vertically depending on your screen resolution):
-    - **Configuration** - used to select your deployment, define system message, and set parameters for interacting with your deployment.
-    - **Chat session** - used to submit chat messages and view responses.
-2. Under **Deployments**, ensure that your gpt-35-turbo-16k model deployment is selected.
-1. Review the default **System message**, which should be *You are an AI assistant that helps people find information.*
-4. In the **Chat session**, submit the following query:
+1. In the **Playgrounds** section (on the left part of AI Foundry portal), select the **Chat** page. The **Chat** playground page consists of a row of buttons and two main panels (which may be arranged right-to-left horizontally, or top-to-bottom vertically depending on your screen resolution):
```

Comment:
Let's change the beginning of this whole line to be "On the left side pane, in the **Playgrounds** section, select the **Chat** page."